modules/phase_field/examples/rigidbodymotion/AC_CH_advection_constforce_rect.i

```diff
@@ -0,0 +1,213 @@
+[Mesh]
```

Comment:
Could you put *a lot more* comments into this example file please. Every block relating to rbm should have a few lines explaining what it is doing.